### ISSUE TITLE
Update react-native-firebase version

### ISSUE THREAD

**Issue by cmcewen** [celo-org/celo-monorepo#2571]
### Expected Behavior

Using 6.x

### Current Behavior

Currently on 5.x

**Comment by jeanregisser** [comment]
Linking to https://github.com/invertase/react-native-firebase/pull/2734 as we might have to make a similar fix for the 6.x release.

**Comment by i1skn** [comment]
The work has started, but for now its decided to put this task on-hold. In this branch https://github.com/celo-org/celo-monorepo/tree/i1skn/migrate-firebase-to-6 I've applied steps described in https://invertase.io/oss/react-native-firebase/migrating-to-v6. Keep in mind, that it, probably, won't even build in its current stage. At the moment of writing v6 does not have `notifications` module (https://github.com/invertase/react-native-firebase/issues/2566#issuecomment-562700412), so I've commented the code related to it. After comparison of the alternatives it was decided to use https://github.com/wix/react-native-notifications for notifications.
